src/app/files-inserter/inbox-linshare-files-inserter.controller.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+angular.module('linagora.esn.unifiedinbox.linshare')
+  .controller('inboxLinshareFilesInserterController', inboxLinshareFilesInserterController);
+
+function inboxLinshareFilesInserterController(onInsert) {
```

Comment:
explicit dependency for onInsert?

Reply:
There'll be another PR to handle explicit dependencies & translation issues.